### DISCUSSION TITLE
Batch processing

### DISCUSSION THREAD

**Discussion by Reginald-L** [lllyasviel/Fooocus#3363] [Q&A]
I am attempting to batch process images using Fooocus. (not one by one) Specifically, I have a batch of image tensors with the shape (B, C, H, W), where B is 8. However, I encounter an error during the sampling process.

![image](https://github.com/user-attachments/assets/51b37037-86e5-4a76-8763-dc34387f2584)
![image](https://github.com/user-attachments/assets/35efbbc3-f113-41e6-85bd-a5685b217a93)

but when B = 1, the first dimentation of tensor h is still 2 times of the tensor inpaint_head_feature

![image](https://github.com/user-attachments/assets/6a24d489-0f3f-449e-a6c9-db1295b5bd9c)

**Comment by mashb1t** [comment]
So you're trying to modify the code to support parallel inpainting for 8 images at once?
Keep in mind that there might be pitfalls, see https://github.com/lllyasviel/Fooocus/discussions/2292.
If you provide the code of what you're trying to do (e.g. as gist) then this can be debugged by the community.
